tests/expr_and_series/first_test.py

```diff
@@ -69,7 +71,7 @@ def test_first_expr_lazy_with_columns(
     expected: PythonLiteral,
     request: pytest.FixtureRequest,
 ) -> None:
-    if any(x in str(constructor) for x in ("pyarrow_table", "pandas")):
+    if any(x in str(constructor) for x in ("pyarrow_table", "pandas", "modin")):
```

Comment:
cudf will suffer from the same problem:

```suggestion
    if any(x in str(constructor) for x in ("pyarrow_table", "pandas", "modin", "cudf")):
```